geo/src/algorithm/coords_iter.rs

```diff
@@ -447,6 +447,50 @@ impl<T: CoordNum> CoordsIter for Geometry<T> {
     }
 }
 
+// ┌──────────────────────────┐
+// │ Implementation for Array │
+// └──────────────────────────┘
+
+impl<const N: usize, T: CoordNum> CoordsIter for [Coord<T>; N] {
+    type Iter<'a> = iter::Copied<slice::Iter<'a, Coord<T>>> where T: 'a;
```

Comment:
Is there a way to not copy each item? It seems like you don't get much of a benefit by implementing this on slices and arrays if you're still copying every coordinate?

Reply:
`CoordsIter` works with `Coord` items so I don't think there's a way around copying each coordinate for now. The copying of `Coord` should be really cheap compared to allocating a `Vec` though, which is the cost I'm trying to avoid.

Reply:
Oh sorry I had a rust brain-fart and forgot copied != cloned 🙈 . Ignore me!

Reply:
🙂 all good!